docs/FAQs.md

```diff
@@ -0,0 +1,158 @@
+# FAQ'S
+Contents of this document.
+* [General](#general)
+* [Deployment and config](#deployment-and-config)
+* [API](#api)
+* [Handler](#handler)
+* [Model-archiver](#model-archiver)
+
+## General
+Relevant documents.
+- [Torchserve readme](https://github.com/pytorch/serve#torchserve)
+
+### Does Torchserve API's follow some REST API standard?
+Torchserve API's are compliant with the [OpenAPI specification 3.0](https://swagger.io/specification/).
+
+### How to use Torchserve in production?
+Depending on your use case, you will be able to deploy torchserve in production using following mechanisms.
+> Standalone deployment. Refer https://github.com/pytorch/serve/docker or https://github.com/pytorch/serve/docs/README.md
+> Cloud based deployment. Refer https://github.com/pytorch/serve/kubernetes https://github.com/pytorch/serve/cloudformation
+
+
+### What's difference between Torchserve and a python web app using web frameworks like Flask, Django?
+Torchserve's main purpose is to serve models via http REST APIs , Torchserve is not a Flask app and it uses netty engine for serving http requests.
+
+Relevant issues: [[581](https://github.com/pytorch/serve/issues/581),[569](https://github.com/pytorch/serve/issues/569)]
+
+### Are there any sample Models available?
+Various models are provided in Torchserve out of the box. Checkout out Torchserve [Model Zoo](https://github.com/pytorch/serve/blob/master/docs/model_zoo.md) for list of all available models. You can also check out the [examples](https://github.com/pytorch/serve/tree/master/examples) folder.
+
+### Does Torchserve has Windows Support?
+Currently, Torchserve supports Windows only via WSL(Windows Subsystem for Linux). The native Windows support will be added in upcoming releases.
+
+### Can I do streaming service with Torchserve like streaming speech recognition?
+Torchserve currently supports only inference through HTTP 1.0 - Request / Response style.
+
+### Is it possible to deploy model other than Pytorch framework?
+Yes it is possible to deploy model with custom python handler or you can use [AWS Multi Model Server](https://github.com/awslabs/multi-model-server) a similar model serving framework.
```

Comment:
@shivamshriwas Please change this. TorchServe is dedicated for PyTorch

Reply:
Done.